Fix ValueError in Question-number user input: stackexchange

NamamiShanker opened the pull request:
## Related Issue
- CLI had a runtime error whenever user input a string instead of number at question number input prompt

Closes: #33 


## Describe the changes you've made
Added an error handling code at user input prompt for dealing with string input instead of numerical. I am using the already implemented SearchError class.


## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.

## Screenshots

|        Original         |          Updated           |
| :---------------------: | :------------------------: |
| **![before](https://user-images.githubusercontent.com/26577306/115139868-0bc2a480-a052-11eb-8c03-c7eee225c8e8.png)** | <b> ![after](https://user-images.githubusercontent.com/26577306/115139877-1715d000-a052-11eb-8514-4b08e020ce6b.png)
 </b> |

NamamiShanker commented:
I mentioned I have used the already implemented SearchError class. Should I create another different class?

GouravSardana commented:
Does quit works ?

NamamiShanker commented:
@GouravSardana Yes it works. But it was working before too. I tested the new code on ubuntu, windows and mac and it was working fine on all three.

NamamiShanker commented:
Hey, @GouravSardana @anshsrtv I wanted your opinion on something. Currently, the get_ans() function is too big, it is responsible for fetching as well as printing the questions as well as the answers. Should we split it into two different modules.. first one will receive question_id and fetch the corresponding answer details. The second one would be responsible for printing them. 

This will probably help when Reddit support is added.

GouravSardana commented:
@NamamiShanker Yup, We can split that into two parts

anshsrtv reviewed (changes requested):
Nicely pointed out! I don't think it will just be a `ValueError`, please check this too.

![image](https://user-images.githubusercontent.com/49693160/115248427-4d804780-a145-11eb-9bb1-df29578cb055.png)

NamamiShanker commented:
@anshsrtv I have fixed this issue with the new PR #87. I changed the entire questions panel functionality. I will close this PR now. Please review the other PR #87.
